### PR TITLE
remove custom config parsing logic

### PR DESCRIPTION
**Pull request by willnorris**
This builds on #5, and should only be merged after that PR.

### CODE REVIEW

**Comment by willnorris** [comment]
I've also done an end-to-end test to confirm this works: https://github.com/willnorris/twitter-test/runs/3098044580?check_suite_focus=true

This is using a repolinter config that extends another config: https://github.com/willnorris/twitter-test/blob/main/.github/repolinter.yaml

(For future reference, I wonder if there is a way to make this testing easier, without having to build and check-in the dist/* files, build and publish a new Docker image, and then updating the action.yml to point to that new image.  An ideal developer workflow would be for all of that to happen as part of the action flow, but we wouldn't want that for normal production use.  Worth thinking about how hard that would be; there must be prior art here)

File: __tests__/getConfig.test.ts [hunk 1]
```diff
@@ -1,121 +0,0 @@
-import getConfig from '../src/getConfig'
```
Comment (prototypicalpro) [inline __tests__/getConfig.test.ts:1]: I have to admit it hurts just a bit seeing all these tests get deleted, but it's for a good cause.

File: README.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ The original version of this tool was created by [New Relic](https://github.com/
     # Defaults to $GITHUB_WORKSPACE.
     directory: ''
 
-    # A path to the JSON/YAML Repolinter ruleset to use, relative to the workflow
-    # working directory (i.e. under `$GITHUB_WORKSPACE`).
+    # A path to the JSON/YAML Repolinter ruleset to use, relative to the
+    # `directory` input.
```
Comment (prototypicalpro) [inline README.md:21]: This is technically a breaking change? But it shouldn't be an issue because no one uses the `directory` parameter IIRC.

**Review by prototypicalpro — APPROVED** [review]
LGTM! As I mentioned before just make sure to add a prefix to the commits. You could also move the `directory` change into it's own commit so it gets it's own item in the changelog, but it's probably not that important.

File: README.md (hunk 2 above)
Comment (willnorris) [reply README.md:21]: So, I was wondering about that too... is anyone *actually* using this in a way that is going to break.  I included the `BREAKING CHANGE` text in my commit, which I think will cause semantic-release to bump the major version.  But do we actually want that here?

**Comment by prototypicalpro** [comment]
> (For future reference, I wonder if there is a way to make this testing easier, without having to build and check-in the dist/* files, build and publish a new Docker image, and then updating the action.yml to point to that new image. An ideal developer workflow would be for all of that to happen as part of the action flow, but we wouldn't want that for normal production use. Worth thinking about how hard that would be; there must be prior art here)

What I've done in the past is edited the `action.yml` to change `runs.using` to `Dockerfile` (or `node12` if none of the ruby dependencies are needed) which will build/run the action on the fly--this still isn't great as Docker builds take forever, but it saves a few steps. I've also tried local testing using https://github.com/nektos/act, but I think it was missing some capabilities that the action needs (maybe not true anymore).

File: README.md (hunk 2 above)
Comment (prototypicalpro) [reply README.md:21]: Naw definitely not, it'll create a new major version tag and we'll have to switch to `todogroup/repolinter-action@v2`. Maybe tag it as a feature or a fix?

My bad for not reading the extended description btw, I forgot it existed.

**Pull request merged**